docs/changes/io.ascii/14365.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Fix an issue in the `io.ascii` QDP format reader to allow lower-case commands in the
```

Comment:
Yeah, we avoid "live links" in the change log because it becomes static and we don't want to "change history" when links break in the future.

```suggestion
Fix an issue in the ``io.ascii`` QDP format reader to allow lower-case commands in the
```